docs/user_guide/introduction.rst

```diff
@@ -0,0 +1,29 @@
+.. _introduction:
+
+Introduction
+===============================================================================
+
+The main purpose of this user guide is to help the user understand and use
+(modify or otherwise) the HDL resources provided by `Analog Devices, Inc.`_,
+and to provide advice and instructions for using these resources.
+These resources are found on the GitHub, the :git-hdl:`/`.
+
+After reading this guide, the user should be able to build a specific project 
+test bench from the :git-testbenches:`/` and be able to modify it (if so
+desired). Furthermore, all ADI developed and supported IPs are presented in detail.
+
+At the same time, this user guide does not intend to be a guide for any third
```

Comment:
```suggestion
The main purpose of this user guide is to help the user understand and use
(modify or otherwise) the HDL resources provided by `Analog Devices, Inc.`_,
and to provide advice and instructions for using these resources.
These resources are found on GitHub at the :git-hdl:`/` and :git-testbenches:`/`.
This user guide complements the :ref-hdl:`HDL User Guide <user_guide>` and the
user should read it first before proceeding.

After reading these guides, the user should be able to build a specific project
testbench from the :git-testbenches:`/` and be able to modify it (if so
desired). Furthermore, all ADI developed and supported IPs are presented in detail.

At the same time, the user guides are not intended to be a guide for any third
```